Makefile

```diff
@@ -1,59 +1,149 @@
+# Project variables
 NAME        := terraform-docs
 VENDOR      := segmentio
 DESCRIPTION := Generate docs from Terraform modules
 MAINTAINER  := Martin Etmajer <metmajer@getcloudnative.io>
 URL         := https://github.com/$(VENDOR)/$(NAME)
 LICENSE     := MIT
 
-VERSION     := $(shell cat ./VERSION)
+# Repository variables
+PACKAGE     := github.com/$(VENDOR)/$(NAME)
 
-GOBUILD     := go build -ldflags "-X main.version=$(VERSION)"
-GOPKGS      := $(shell go list ./... | grep -v /vendor)
+# Build variables
```

Comment:
Can you please sort the variables in each block where possible?

Reply:
+1